src/collections/_documentation/platforms/php/index.md

```diff
@@ -382,12 +427,19 @@ use Sentry\State\Hub;
 
 $options = ['dsn' => '___PUBLIC_DSN___'];
 
-$spool = new MemorySpool();
-$transport = new SpoolTransport($spool);
+
 $httpTransport = new HttpTransport($options, HttpAsyncClientDiscovery::find(), MessageFactoryDiscovery::find());
 
+$transportFactory = new class implements TransportFactoryInterface {
+    public function create(\Sentry\Options $options): \Sentry\Transport\TransportInterface
+    {
+        $spool = new MemorySpool();
```

Comment:
Since the spool is the class that users should interact with to flush the events, it should be instantiated outside of the factory